types/stylelint/index.d.ts

```diff
@@ -860,11 +860,24 @@ declare namespace stylelint {
 		quiet?: boolean;
 		quietDeprecationWarnings?: boolean;
 		validate?: boolean;
+		suppressAll?: boolean;
+		suppressLocation?: string;
+		suppressPrune?: boolean;
+		suppressRule?: string[];
 	};
 
 	/** @internal */
 	export type FixMode = 'lax' | 'strict';
 
+	/** @experimental */
+	export type SuppressedViolations = {
+		[ruleName: string]: {
+			[message: string]: {
```

Comment:
question

Shouldn't this be
```suggestion
		[path: string]: {
			[ruleName: string]: {
```
?